node/network/statement-distribution/src/lib.rs

```diff
@@ -210,16 +270,35 @@ impl<R: rand::Rng> StatementDistributionSubsystem<R> {
 			})) => {
 				let _timer = metrics.time_active_leaves_update();
 
-				// TODO [now]: vstaging should handle activated first
-				// because of implicit view.
+				// vstaging should handle activated first because of implicit view.
```

Comment:
This comment is wrong actually (my fault) - the internal logic of `handle_active_leaves_update` does this internally. We can clean up this section as a result and make only one call to `handle_active_leaves_update`.